libs/dh/esett/feature-outgoing-messages/src/lib/details.component.ts

```diff
@@ -174,18 +174,11 @@ export class DhOutgoingMessageDetailsComponent {
   downloadXML(documentType: 'message' | 'receipt') {
     const fileName = `eSett-outgoing-${this.outgoingMessage()?.documentId}-${documentType}`;
```

Comment:
nitpick: one last thing

```suggestion
    const fileName = `DataHub-eSett-outgoing-${this.outgoingMessage()?.documentId}-${documentType}`;
```